rhscl.yaml

```diff
@@ -74,6 +101,33 @@ templates:
     envs:
       - "BASH_ENV        /etc/profile.d/cont-env.sh"
     cmd: ["container-usage"]
+    devel_libs: |
+      \
+      autoconf \
+      automake \
+      bsdtar \
+      curl-devel \
+      epel-release \
+      gcc-c++ \
+      gdb \
+      gettext \
+      libxml2-devel \
+      libxslt-devel \
+      lsof \
+      make \
+      mysql-devel \
+      mysql-libs \
+      openssl-devel \
+      postgresql-devel \
+      procps-ng \
+      scl-utils \
+      sqlite-devel \
+      tar \
+      unzip \
+      wget \
+      which \
+      yum-utils \
+      zlib-devel \
```

Comment:
@hhorak 'find' is pretty common thing and already bundled into the 'centos" image which causes inconsistency with RHEL image. Actually I think 'findutils' should be part of the 'RHEL7' image itself. @vpavlin?